### ISSUE TITLE
[Simplified Chinese] Keyboard Compatibility

### ISSUE THREAD

**Issue by JinMokai** [w3c/wai-translations#258]
### Page URL

https://www.w3.org/WAI/perspective-videos/keyboard/

### Instructions

- [X] I have read the information on [Translating WAI Documents](https://www.w3.org/WAI/about/translating/)
- [X] Before starting the translation, I will wait for confirmation that the resource is ready for translation.

### Link to original file in GitHub

https://github.com/w3c/wai-website/edit/main/collections/_perspective-videos/keyboard.md

**Comment by JinMokai** [comment]
Why is the translated keyboard page article not appearing in the preview link?

**Comment by remibetin** [comment]
@JinMokai Thanks for volunteering to translate this resource. Are you planning on translating all "Perspective Videos" pages?

**Comment by JinMokai** [comment]
> @JinMokai Thanks for volunteering to translate this resource. Are you planning on translating all 10 "Perspective Videos" pages (+ index page)?

Yes, I will translate them, but I am going to translate the resources you gave me in the email first.

**Comment by JinMokai** [comment]
Hello @remibetin 
 The translation is ok for me, need review.🧨

**Comment by remibetin** [comment]
This translation is ready for review! 🎉

Reviewers are welcome to add their comments and collaborate with the translator in the related Pull Request: https://github.com/w3c/wai-website/pull/960

Guidance: https://www.w3.org/WAI/about/translating/guides/review-translation/

**Comment by remibetin** [comment]
Hello @JinMokai,

Thanks for translating this resource. This page features a video that does not have Simplified Chinese subtitles and descriptions yet. Would you be willing to create them? Since you have already translated the text transcript, it should be rather straightforward.

A guide is available — [How to Create Translated Video Subtitles and Descriptions](https://www.w3.org/WAI/about/translating/resources/subtitles/) — and I am happy to provide further guidance.

The relevant files are located at https://github.com/w3c/wai-website-data/tree/master/wai-videos/perspective-videos

**Comment by JinMokai** [comment]
> Hello @JinMokai,
> 
> Thanks for translating this resource. This page features a video that does not have Simplified Chinese subtitles and descriptions yet. Would you be willing to create them? Since you have already translated the text transcript, it should be rather straightforward.
> 
> A guide is available — [How to Create Translated Video Subtitles and Descriptions](https://www.w3.org/WAI/about/translating/resources/subtitles/) — and I am happy to provide further guidance.
> 
> The relevant files are located at [w3c/wai-website-data@`master`/wai-videos/perspective-videos](https://github.com/w3c/wai-website-data/tree/master/wai-videos/perspective-videos)

@remibetin I have already translated the content of this video. Thank you for your guidance. You can find the translation here: w3c/wai-website-data#183

**Comment by remibetin** [comment]
This translation is ready for review! 🎉

Reviewers are welcome to add their comments and collaborate with the translator in the related Pull Requests:
- Page content: https://github.com/w3c/wai-website/pull/960
- Subtitles: https://github.com/w3c/wai-website-data/pull/183

@JinMokai Please note that the review of this translation may take longer, since it contains sensitive accessibility terminology.

Guidance: https://www.w3.org/WAI/about/translating/guides/review-translation/

cc @ruoxiran @heybran
